### ISSUE TITLE
Allow Rails 6 beta2

### ISSUE THREAD

**Issue by n-rodriguez** [fphilipe/premailer-rails#235]
Hi there!

It could be nice to test this gem with Rails 6 beta 2 : 

```sh
Bundler could not find compatible versions for gem "actionmailer":
  In gems.rb:
    letter_opener_web was resolved to 1.3.4, which depends on
      actionmailer (>= 3.2)

    premailer-rails was resolved to 1.10.2, which depends on
      actionmailer (>= 3, < 6)

    rails (= 6.0.0.beta2) was resolved to 6.0.0.beta2, which depends on
      actionmailer (= 6.0.0.beta2)
```

Thank you!

**Comment by fphilipe** [comment]
This shouldn't be a problem. Note that premailer-rails currently requires actionmailer < 6, but by definition of SemVer 6.0.0.beta2 is lower than 6.

See the latest build of master against master of rails: https://travis-ci.org/fphilipe/premailer-rails/jobs/495313952#L562

Also, see this previous issue about rails 6 beta: https://github.com/fphilipe/premailer-rails/issues/231

Thus, try removing your `Gemfile.lock` and rerunning `bundle install`.

**Comment by n-rodriguez** [comment]
> Thus, try removing your `Gemfile.lock` and rerunning `bundle install`.

It works! Thank you!

**Comment by knagode** [comment]
```
 In Gemfile:
    premailer-rails was resolved to 1.10.2, which depends on
      actionmailer (< 6, >= 3)

    rails was resolved to 6.0.0.beta2, which depends on
      actionmailer (= 6.0.0.beta2)`
```

I am also getting this. I removed `Gemfile.lock`, ran `bundle install` but I still see this error. Any other idea @fphilipe or @n-rodriguez ?

**Comment by knagode** [comment]
Oh .. :) Thanks for noticing that I am in wrong repo. Will do as you said
